setup.py

```diff
@@ -16,4 +16,8 @@
     entry_points={
         "console_scripts": ["hcpi=HCPInterface.cli.base:root"],
     },
+    data_files=[('config', ['config.ini']),
+                ('reference', ['reference/covidMetadataAllowedValues.csv', 'reference/covidMetadataTemplate.csv']),
+                ('testdata', ['tests/data/test_reads_R1.fasterq','tests/data/test_reads_R1.fastq.gz','tests/data/test.json'])
```

Comment:
Just a thought. Is this dataset completely cleaned from any human DNA? So we don't end up in a situation where we are using any identifiable data in our tests.

Reply:
Yea its just random garbo that looks like fastq